src/index.ts

```diff
@@ -703,11 +703,51 @@ class ServerlessCustomDomain {
         }
     }
 
+    /**
+     * Gets value by name from cloudformation exports
+     */
+    public async getImportValue(importValueName, nextToken = null): Promise<string> {
+      const params = nextToken ? { NextToken: nextToken } : {};
```

Comment:
We have a function called `getAWSPagedResults` in utils that could be used to page through the import values instead